extensions/azure/package.json

```diff
@@ -0,0 +1,128 @@
+{
+	"name": "azure",
+	"displayName": "azure",
```

Comment:
Name :-)

Reply:
OK I've renamed to **Azure (Core)** as this is the closes I can get to reality. I expect additional Azure-related functions to go here, but you're right that future data extensions likely want to contribute to this core extension (e.g. define connectors and browse capability for new data resources dynamically).